junit-jupiter-params/junit-jupiter-params.gradle.kts

```diff
@@ -20,7 +20,7 @@ dependencies {
 	compileOnlyApi(libs.apiguardian)
 	compileOnly(libs.jspecify)
 
-	shadowed(libs.univocity.parsers)
+	implementation(libs.fastcsv)
```

Comment:
We should shadow FastCSV to avoid classpath conflicts in case user code uses a different version of FastCSV.